docs/development/i18n.rst

```diff
@@ -0,0 +1,48 @@
+Internationalization AKA i18n
+=============================
+
+The code, templates and javascript user visible strings must all be wrapped with a `gettext`_ functions to be substituted at runtime with the equivalent localized string. These function names are used as markers to collect the strings to be translated and store them into the ``securedrop/translations/messages.pot`` file. For each language to be translated, a directory is created such as ``securedrop/translations/fr_FR`` and populated with files derived from ``securedrop/translations/messages.pot``, for translators to work with and for the `gettext`_ substitution at runtime.
+
+The manage.py translate helper
+------------------------------
+
+The `pybabel`_ command line is wrapped into the ``manage.py translate`` helper for convenience. It is designed to be used by developers, to run tests with fixtures and during packaging.
+
+Updating strings to be translated
+---------------------------------
+
+After modifying a string in the code, templates or javascript, the ``securedrop/translations/messages.pot`` file must be updated by running the following command at the root of the git tree, in the development virtual machine::
+
+    ./manage.py --verbose translate --extract-update
+
+The updated ``securedrop/translations/messages.pot`` should then be reviewed and commited. Once merged, the changes will be visible in the `weblate`_ web interface used by translators.
+
+Compiling translations
+----------------------
+
+The `gettext`_ need a compiled file for each language (the ``*.mo`` files). This can be done by running the following command at the root of the git tree, in the development virtual machine::
+
+    ./manage.py --verbose translate --compile
+
+
+Verifying translations
+----------------------
+
+After a translation is compiled, the web page in which it shows can be verified visually by navigating to the corresponding state from ``http://localhost:8080`` for the source interface or ``http://localhost:8081`` for the journalist interface after running the following::
+
+     ./manage.py run
+
+An easier way is to generate screenshots for each state in each known language with::
+
+     pytest -v tests/pages-layout
```

Comment:
In this case for clarity one can use `$ pytest -v tests/pages-layout` to clearly denote that this is a command and the text that follows is output of that command